staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -79,6 +79,10 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		ctx := scope.ContextFunc(req)
 		ctx = request.WithNamespace(ctx, namespace)
 
+		// TODO: this is NOT using the scope's convertor [sic]. That seems
+		// like a potential extremely hard to find bug. Already some
+		// tests set this converter but apparently not the scope's
+		// unsafeConvertor.
```

Comment:
at a glance, it appears the passed-in convertor might be tailored to the resource being patched (it comes from the mapping looked up for that specific resource), whereas the scope's convertor is obtained from the API-group-wide convertor. Given that we're already using the scope's unsafe convertor, I doubt the second one is necessary, but if it was going to make a difference, I'd check patching a subresource where the API group is different than the containing resource's group (like scale subresources). Agree a follow-up should resolve this and probably collapse on the scope unsafe converter (verifying the resulting `schemaReferenceObj` is identical)